vertical-pod-autoscaler/hack/vpa-process-yamls.sh

```diff
@@ -41,9 +41,6 @@ fi
 
 ACTION=$1
 COMPONENTS="vpa-v1-crd-gen vpa-rbac updater-deployment recommender-deployment admission-controller-deployment"
-case ${ACTION} in
-delete|diff|print) COMPONENTS+=" vpa-beta2-crd" ;;
```

Comment:
I think we still want to delete the old CRD

Reply:
> so you mean to mention the old CRDs until we delete them in future releases? As I mentioned in this [comment](https://github.com/kubernetes/autoscaler/issues/5832#issuecomment-1576567132). Or Did I miss something here?


@jbartosik, could you please take a look?

Reply:
I mean if someone has beta CRD in cluster and runs the script to delete VPA we should delete beta CRD

Reply:
> I mean if someone has beta CRD in cluster and runs the script to delete VPA we should delete beta CRD

yeah, you are right, it should delete the beta CRD if it would be in the cluster.